src/vochain/vochain.proto

```diff
@@ -207,6 +221,58 @@ message RegisterKeyTx {
         string weight = 5;   // Weight to delegate to newKey
 }
 
+message MintTokensTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	bytes to = 3;
+	bytes value = 4;
+}
+
+message SendTokensTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	bytes from = 3;
+	bytes to = 4;
+	bytes value = 5;
+}
+
+message SetTransactionCostsTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	uint64 value = 3;
+}
+
+message SetAccountInfoTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	string infoURI = 3;
+}
+
+message AddDelegateForAccountTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	bytes delegate = 3;
+}
+
+message DelDelegateForAccountTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	bytes delegate = 3;
+}
+
+message CollectFaucetTx {
+	TxType txType = 1;
+	bytes nonce = 2;
+	FaucetPayload payload = 3;
+	bytes payloadSignature = 4;
+}
+
+message FaucetPayload {
+	bytes nonce = 1;
```

Comment:
This confuses me a bit. Then we have 3 types of nonce: `bytes`, `uint32` and `uint64`, does not make more sense to just use the same data type everywhere ? Let me know, thank you

Reply:
I replied that before. 

In this specific case the nonce is NOT a sequence number and neither an entropy value. It's a unique identifier that do not require to be sequenced and needs to be stored in the state. Maybe we can callit `identifier` instead of nonce in order to avoid confusion.